website/blog/2023-2-4-where-to-store-corso.md

```diff
@@ -0,0 +1,165 @@
+for a variety of different use cases and
+Corso can leverage a number of them, but not all, to reduce the cost of storing
+data in the cloud.
+
+By default, Corso works hard to reduce its data footprint.
+It will compress and deduplicate data at source to reduce the amount of storage
+used as well as the amount of network
+traffic when writing to object storage.
+Corso also combines different emails, attachments, etc.
+into larger objects to make it more cost-effective by reducing the number of API
+calls and increasing network throughput as well as making Corso data eligible an
+cost-effective for some of the other
+storage classes described below.
+
+Stepping away from the default S3 offering (S3 Standard), S3 offers a number of
+different Glacier (cheap and deep) storage classes that can help to further
+reduce the cost for backup and archival workloads.
+Within the storage classes, Corso today supports Glacier Instant Retrieval but,
+because of user responsiveness and metadata requirements, not the other Glacier variants.
+
+Digging into how Corso works, it will often delete and rewrite the data and
+metadata indices within a repository.
+That’s how it can offer you great detail into your backups,
+like in this video where I look at [individual backed-up emails](https://youtu.be/mweAUDhUE7I) from my exchange server.
+As a result of these frequent reads and writes, and the way Corso garbage
+collects deleted backups, AWS S3 Glacier Deep Archive or Flexible Retrieval,
+with several minutes or longer request latency, won’t work with Corso,
+full stop.
+
+However, Glacier Instant Retrieval (IR) is a good candidate to store the objects
+that represent the backup data but not the various indexes needed to maintain the
+repo.
+Glacier Instant Retrieval should provide the best price performance for a  backup workload as backup data blobs are
+typically written once, with occasional re-compacting, and read infrequently in the case of restore.
+One should note that recommendations such as these are always workload dependant and should be verified for your use
+case.
+For example, we would not recommend Glacier IR if you are constantly testing large restores or have heavy churn in your
+backups and limited retention.
+
+For most typical backup workloads (write mostly, read rarely), Glacier Instant Retrieval should work just fine and
+deliver the best price-performance ratio.
+Unlike a naive backup solution and as mentioned above, Corso efficiently packs smaller backup items (e.g., multiple
+emails) into a larger object.
+This makes Corso’s backup data eligible for S3 Intelligent Tiering where infrequently accessed objects can be moved into
+Glacier IR.
+However, if you have a typical backup workload, Corso also supports putting objects containing backup data directly into
+the Glacier IR storage class without waiting for a “cooling off”
+time spent in the more expensive S3 Standard storage class.
```

Comment:
This section on Intelligent Tiering seems weirdly placed and the transitions from the Glacier IR one is too abrupt. 

I would consider having too distinct sections with headings with recommendations to consider in order of priority
* Glacier IR
* S3 Intellignet tiering

It would be much better if there is content along the lines pros and cons for each or at least examples of what workloads are good for both. 

For Glacier IR you sort of have that - typical backup workload.

When should I consider S3 Intelligent Tiering? - maybe when I not sure about the relative frequency of restores and  my retention policy. 

As is the tiering section is mentioned but it is unclear why I would care because then it says use Glacier IR anyway. 

If you go with the subsection approach, you can also include when to use S3 Standard - it is already in the text - performing testing with large repos where you are churning data rapidly and performing frequent test restores, etc

This is probably more feedback than you asked for so you can chose how to use it

Reply:
Georgi, I agree this is a lot that gets backed in here and the transitions are awkward. I guess it evolved this way, with contributions from me, from wanting to position this as Glacier in the context of Corso. Your suggestions would improve the article but it is a significant rework (and maybe even its own article?). I will defer to @serverless-mom about what she wants to do.

Reply:
I think we cut everything about intelligent tiering and perhaps write a follow up piece on 'what S3 storage class is right for backups,' I can get a new version drafted and sent over. Since it's really not intended for prose reviews, Github is not the best place for large rewrites, I'll take this back to Notion and work from there

Reply:
resolved by rewrites. I think a 'choosing the best AWS storage class for your backups' is a worthwhile blog post topic, will draft